### DISCUSSION TITLE
v1.0.0-rc5 自定义音频发送失效

### DISCUSSION THREAD

**Discussion by yuntaowu2000** [Mrs4s/go-cqhttp#2019] [Q&A] [answered]
使用[custom 协议](https://github.com/Mrs4s/go-cqhttp/blob/v1.0.0-rc5/coolq/cqcode.go#L720)，发送的信息为：
`[CQ:music,type=custom,subtype=163,url=http://trails-game.com,audio=https://trails-game.com/wp-content/uploads/2021/07/renne.wav,title=test]`。尝试过各类subtype，但是点击后不会播放音频，只会跳转到url页面。

**Comment by ilihu** [comment, marked as answer]
POST  /send_msg
```
 {
    "group_id": "123123",
    "message_type": "group",
    "message": {
        "type": "record",
        "data": {
            "file": "https://trails-game.com/wp-content/uploads/2021/07/renne.mp3"
        }
    }
}
```
请转成mp3食用
